tests/cypress/integration/digitall.home.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { siteHomePage } from '../page-object/site.home.page'
+
+describe('Sitemap 4.0.0', () => {
+    it('Create sitemap for a site', function () {
+        siteHomePage
+            .goTo()
+            .editPage('Digitall')
+            .clickOnSitemap()
+            .validateSucessMessage()
+            .clickBack()
+            .publishSite()
+            .clickPublishAll()
+            .flushCache()
+
+        // fetch the sitemap content
+        cy.request('sites/digitall/sitemap.index.xml').then((response) => {
```

Comment:
Is there a way to get a XML fixture then do a comparison?
So we have a more descriptive check?

Reply:
not familiar with such tools or libraries